src/howitz/utils.py

```diff
@@ -52,3 +52,31 @@ def set_correct_timezone(dt: datetime):
 def date_str_without_timezone(dt: datetime):
     dt_aware = set_correct_timezone(dt)
     return dt_aware.strftime("%Y-%m-%d %H:%M:%S")
+
+
+# Implementation copied from curitz
+def shorten_downtime(td: timedelta):
+    """
+    Shortens downtime to single greatest applicable timedelta unit (days, hours, minutes, seconds).
+    :param td: downtime as `datetime.timedelta`
+    :return: formatted downtime as string
+    """
+    if td.days > 0:
+        return "{:2d}d".format(td.days)
+    if td.seconds < 60:
+        return "{:2d}s".format(td.seconds)
+    if td.seconds / 60 < 60:
+        return "{:2.0f}m".format(td.seconds / 60)
+    if td.seconds / 60 / 60 < 60:
+        return "{:2.0f}h".format(td.seconds / 60 / 60)
+
+
+def calculate_event_age_no_seconds(opened: datetime):
+    """
+    Calculate event age and remove seconds.
+    :param opened: event opened time as `datetime.datetime`
+    :return: event age without seconds as string
+    """
+    age = datetime.now(timezone.utc) - opened
+    age_formatted = str(age)[:-10]
+    return age_formatted
```

Comment:
It's soo tempting making a "tznow" utils-function for `datetime.now(timezone.utc)`.